pilot/cmd/pilot-agent/main.go

```diff
@@ -681,7 +721,8 @@ func init() {
 
 	proxyCmd.PersistentFlags().Uint16Var(&statusPort, "statusPort", 0,
 		"HTTP Port on which to serve pilot agent status. If zero, agent status will not be provided.")
-
+	proxyCmd.PersistentFlags().IntVar(&stsPort, "stsPort", 0,
```

Comment:
what happens if you run this outside gcp?

Reply:
Currently, this does not support other environments. If there is a universal API that we can use in the future, then we can support other environments. For example, GCP and AWS both support the same token API calls. @liminw